content/enterprise_features/sakuli-dashboard-k8s.md

```diff
@@ -0,0 +1,153 @@
+---
+title : "Sakuli Dashboard on Kubernetes"
+date :  2019-09-12T14:16:46+02:00
+weight : 7
+---
+
+# Sakuli Dashboard on Kubernetes
+
+Once you obtained a Sakuli Enterprise license your docker-user will be granted access to the private Sakuli dashboard image.
+
+## Prerequisites
+To setup the Sakuli dashboard on your Kubernetes cluster, it is required to import the image from 
+`taconsol/sakuli-dashboard`. To achieve this, you have to create a docker-registry secret with your `<docker-username>` and
+`<docker-password>` and add it to your service account to authenticate on docker.io during build.
+
+{{<highlight bash>}}
+kubectl create secret docker-registry dockerhub-sakuli-secret \
+ --docker-server=docker.io \
+ --docker-username=<docker-username> \
+ --docker-password=<docker-password> \
+ --docker-email=unused
+ 
+kubectl patch serviceaccount default -p '{"imagePullSecrets": [{"name": "dockerhub-sakuli-secret"}]}'
+{{</highlight>}}
+ 
+Now it is possible to import the image from the secured registry.
```

Comment:
Maybe we should delete this line.
Since we added the `imagePullSecret` to the default service account there's actually nothing to do here, although it might make you think there is.